src/main/resources/application.yml

```diff
@@ -9,6 +9,10 @@ spring:
       database: iexec
       host: ${MONGO_HOST:localhost}
       port: ${MONGO_PORT:13202}
+      auto-index-creation: true
+  mvc:
+    pathmatch:
+      matching-strategy: ant_path_matcher # see https://stackoverflow.com/a/70037507/7421571
```

Comment:
Have you played with Spring Doc as https://stackoverflow.com/a/70106133 suggests?

Reply:
No, I preferred to stick with the current matcher.

Reply:
I'm fine with that for now, but I see "Note that this will only work if you are not using Spring Boot's Actuator" while core and worker use actuators.

Reply:
Will it work with the SMS and adapter? We don't use spring-actuator there.

Reply:
It should work, yes. But I could move from SpringFox to SpringDoc, it'll be better.